docs/specification.md

```diff
@@ -79,19 +79,19 @@ Stablecoin contract MUST provide an additional metadata `big_map` for its tokens
 This `big_map` MUST be annotated as `%token_metadata` and can be at any position within
 contract storage. The structure of this `big_map` MUST follow [FA2 requirements](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-12/tzip-12.md#token_metadata-big_map).
 
-# Safelist
+# Transferlist
 
-Before describing the token contract's entrypoints we describe the `Safelist` interface.
+Before describing the token contract's entrypoints we describe the `Transferlist` interface.
 The syntax is: `entrypointName(argumentType)`.
-Required `Safelist` entrypoints:
+Required `Transferlist` entrypoints:
 * `assertTransfers(list (pair (address %from) (list %tos address)))`
   * Checks whether a transfer is permitted from one address to the other one.
   Fails if transfer is prohibited for any item from this list.
-* `assertReceiver(address)`
-  * Fails if address is not whitelisted or it is blacklisted.
 * `assertReceivers(list address)`
   * Fails if any address in the list is not whitelisted or is blacklisted.
 
+The specific interface of transferlist contracts is described in [TZIP-15](https://gitlab.com/tzip/tzip/-/blob/c38f4c2dc9da883d01e85409d34d0015d6ad272a/proposals/tzip-15/tzip-15.md)
```

Comment:
It might be worth adding that we need `UserId` to be an `Address` for the Transferlist contract to be compatible with this contract.